docs/readmes/README.namespaced.md

```diff
@@ -0,0 +1,40 @@
+# Match Pods in the Namespace
+> Available from [1.3.0-mic-1.4.0-nmi release](https://github.com/Azure/aad-pod-identity/releases/tag/1.3.0-mic-1.4.0-nmi)
+
+By default, AAD Pod Identity matches pods to identities across namespaces. To match only pods in the namespace containing `AzureIdentity`, use one of these techniques:
+
+* Attach a `aadpodidentity.k8s.io/Behavior: namespaced` [annotation] to each `AzureIdentity` resource.
+
+    Here is the `AzureIdentity` manifest from the previous step with this annotation added:
+
+    ```yaml
+    apiVersion: "aadpodidentity.k8s.io/v1"
+    kind: AzureIdentity
+    metadata:
+      name: <a-idname>
+      annotations:
+        aadpodidentity.k8s.io/Behavior: namespaced
+    spec:
+      type: 0
+      ResourceID: /subscriptions/<subid>/resourcegroups/<resourcegroup>/providers/Microsoft.ManagedIdentity/userAssignedIdentities/<name>
+      ClientID: <clientId>
+    ```
+
+* Add the `--forceNamespaced` command line argument or set the `FORCENAMESPACED=true` environment variable when starting both the MIC and NMI components.
```

Comment:
We should say doing this will enable forced namespace for all identity/bindings/pod created. Also, I think we should mention even though the force namespace directive is used, the azure assigned identity is always created in default namespace.

Reply:
> Also, I think we should mention even though the force namespace directive is used, the azure assigned identity is always created in default namespace.

Please ignore this.